installation/manifest/validator.go

```diff
@@ -24,21 +25,16 @@ func NewValidator(logger boshlog.Logger) Validator {
 }
 
 func (v *validator) Validate(manifest Manifest, releaseSetManifest birelsetmanifest.Manifest) error {
-	errs := []error{}
+	var errs []error
 
-	cpiJobName := manifest.Template.Name
-	if v.isBlank(cpiJobName) {
-		errs = append(errs, bosherr.Error("cloud_provider.template.name must be provided"))
-	}
-
-	cpiReleaseName := manifest.Template.Release
-	if v.isBlank(cpiReleaseName) {
-		errs = append(errs, bosherr.Error("cloud_provider.template.release must be provided"))
+	// When there is nothing in templates, return an error. It should have a CPI release.
+	if len(manifest.Templates) == 0 {
+		return fmt.Errorf("manifest.Templates cannot be empty and must contain one release")
```

Comment:
Not sure `manifest.Templates` tells them what they need to do since it's not the actual path in the manifest. I'm guessing this is `cloud_provider.template`?

Reply:
Oh, yeah, that is definitely the wrong name to use in that error message.

Does `"either cloud_provider.templates or cloud_provider.template must be provided and must contain at a CPI release"` seem like a good instruction for that error?

Reply:
I've gone ahead and reworded the message. See the diff I posted in the next "conversation".